app/file.php

```diff
@@ -1,2 +1,50 @@
 <?php
 
+declare(strict_types = 1);
+$root = dirname(__DIR__) . DIRECTORY_SEPARATOR;
+define('FILES_PATH', $root . 'files' . DIRECTORY_SEPARATOR);
+
+class File
+{
+    private string $file_name;
+    private string $file_path;
+    private $raw_data;
+    private array $formatted_data;
+    
+    public function __construct(string $file_name, string $file_path = FILES_PATH)
+    {        
+        $this->file_name = $file_name;        
+        
+        $this->file_path = $file_path;        
+    }   
+
+    public function getFilePath(): string
+    {
+        return $this->file_path;
+    }
+
+    public function getFileName(): string
+    {
+        return $this->file_name;
+    }
+
+    public function toBeExtraced(): string | array 
```

Comment:
Maybe a Typo here (toBeExtracted)? Also, is good practice to name functions/methods that do something with a verb in infinitive (e.g. extract).

Reply:
Done